src/java/org/apache/cassandra/db/SystemKeyspace.java

```diff
@@ -255,6 +251,18 @@ private SystemKeyspace()
                 + "PRIMARY KEY ((keyspace_name, columnfamily_name, generation)))")
                 .build();
 
+    private static final TableMetadata SSTableActivity =
+        parse(SSTABLE_ACTIVITY_V2,
+                "historic sstable read rates",
+                "CREATE TABLE %s ("
+                + "keyspace_name text,"
+                + "columnfamily_name text,"
+                + "generation blob,"
```

Comment:
I think this hasn't been addressed yet. By the way, since we are creating a new version of the table, we could rename the `columnfamily_name` column to `table_name`, which is the name that we use in `IndexInfo`, `paxos_repair_history` and `table_estimates`.

Reply:
@jacek-lewandowski I see the new `table_name` column name, which is nice, but we are still missing renaming the `generation` column to `id`.